src/meta.ml

```diff
@@ -195,6 +195,11 @@ let archives name =
 
 let builtins ~stdlib_dir =
   let version = version "[distributed with Ocaml]" in
+  let sep_libraries =
+    List.exists (fun suf ->
```

Comment:
the label is missing (that's what causing the CI failures)

Reply:
Hum, weird I was not getting this earlier but instead I had warning 32 in usexp.mli. Not sure what happened but I fixed this.